mmseg/models/backbones/mae.py

```diff
@@ -0,0 +1,509 @@
+# Copyright (c) OpenMMLab. All rights reserved.import math
+import math
+import warnings
+
+import torch
+import torch.distributed as dist
+import torch.nn as nn
+from mmcv.cnn import build_norm_layer
+from mmcv.cnn.bricks.drop import build_dropout
+from mmcv.cnn.bricks.transformer import FFN
+from mmcv.cnn.utils.weight_init import (constant_init, kaiming_init,
+                                        trunc_normal_)
+from mmcv.runner import BaseModule, ModuleList, _load_checkpoint
+from torch.nn.modules.batchnorm import _BatchNorm
+from torch.nn.modules.utils import _pair as to_2tuple
+
+from mmseg.utils import get_root_logger
+from ..builder import BACKBONES
+from ..utils import PatchEmbed
+
+
+class MAEAttention(BaseModule):
```

Comment:
mae atten might be very similar to beit attn?

Reply:
`qkv bias` in `MAEAttention` is different from that in `BEiTAttention` , so I rewrite the attention module.